flink-connectors/flink-connector-base/src/main/java/org/apache/flink/connector/base/table/sink/options/SinkConnectorOptionsUtils.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.connector.base.table.sink.options;
+
+import org.apache.flink.annotation.Internal;
+import org.apache.flink.connector.base.table.options.TableOptionsUtils;
+
+import java.util.Properties;
+
+/** Interface for classes handling table options for sink connectors. */
+@Internal
+public interface SinkConnectorOptionsUtils extends TableOptionsUtils {
+
+    /**
+     * Method for extracting and wrapping sink specific table options.
+     *
+     * @return Sink specific configuration.
+     */
+    Properties getSinkProperties();
```

Comment:
I am not sure this is really Sink specific... It is just returning some properties, and it is not used by the KDS Async Sink